iOS/GPUImage/nuget/GPUImage.nuspec

```diff
@@ -3,14 +3,14 @@
   <metadata>
     <id>GPUImage</id>
     <title>GPUImage</title>
-    <version>0.1.7.1</version>
+    <version>0.1.7</version>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>An open source iOS framework for GPU-based image and video processing.</description>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
-    <projectUrl>https://go.microsoft.com/fwlink/?linkid=865063</projectUrl>
-    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=864973</licenseUrl>
+    <projectUrl>https://components.xamarin.com/view/GPUImage</projectUrl>
+    <licenseUrl>https://components.xamarin.com/license/gpuimage</licenseUrl>
```

Comment:
needs to use FWLinks -> https://gist.github.com/bholmes/39bad5c2296fe445df9e5bbf9261ab15